docs/extra.css

```diff
@@ -0,0 +1,3 @@
+td {
+    padding: 10px
```

Comment:
This is a bug of readthedocs theme, I'm proposing to fix it [this way](https://github.com/mkdocs/mkdocs/issues/2028#issuecomment-605056544).